windows/ReactNativeBlobUtil/ReactNativeBlobUtil.cpp

```diff
@@ -108,1406 +110,1806 @@ void TaskCancellationManager::Cancel(TaskId taskId) noexcept
+                //m_reactContext.CallJSFunction(L"RCTDeviceEventEmitter", L"emit", [&streamId, &base64Content](React::IJSValueWriter const& argWriter) {
+                //	argWriter.WriteArrayBegin();
+                //	WriteValue(argWriter, streamId);
+                //	argWriter.WriteObjectBegin();
+                //	React::WriteProperty(argWriter, "event", "data");
+                //	React::WriteProperty(argWriter, "detail", base64Content);
+                //	argWriter.WriteObjectEnd();
+                //	argWriter.WriteArrayEnd();
+                //	});
+                m_reactContext.CallJSFunction(L"RCTDeviceEventEmitter", L"emit", streamId,
+                    winrt::Microsoft::ReactNative::JSValueObject{
+                        {"event", "data"},
+                        {"detail", winrt::to_string(base64Content)},
+                    });
+            }
+            else
+            {
+                // TODO: Sending events not working as necessary with writers
+                std::string utf8Content{ winrt::to_string(Cryptography::CryptographicBuffer::ConvertBinaryToString(BinaryStringEncoding::Utf8, readBuffer)) };
+                if (usedEncoding == EncodingOptions::ASCII)
+                {
+
+                    //std::string asciiContent{ winrt::to_string(utf8Content) };
+                    std::string asciiContent{ utf8Content };
+                    // emit ascii content
+                    m_reactContext.CallJSFunction(L"RCTDeviceEventEmitter", L"emit", streamId,
+                        winrt::Microsoft::ReactNative::JSValueObject{
+                            {"event", "data"},
+                            {"detail", asciiContent},
+                        });
+                }
+                else
+                {
+                    //emit utf8 content
+                    m_reactContext.CallJSFunction(L"RCTDeviceEventEmitter", L"emit", streamId,
+                        winrt::Microsoft::ReactNative::JSValueObject{
+                            {"event", "data"},
+                            {"detail", utf8Content},
+                        });
+                    //m_reactContext.CallJSFunction(L"RCTDeviceEventEmitter", L"emit", [&streamId, &utf8Content](React::IJSValueWriter const& argWriter) {
+                    //	WriteValue(argWriter, streamId);
+                    //	argWriter.WriteObjectBegin();
+                    //	React::WriteProperty(argWriter, "event", L"data");
+                    //	React::WriteProperty(argWriter, "detail", utf8Content);
+                    //	argWriter.WriteObjectEnd();
+                    //	});
+                }
+            }
+            // sleep
+            if (tick > 0)
+            {
+                std::this_thread::sleep_for(std::chrono::milliseconds(static_cast<int64_t>(tick)));
+            }
+        }
+    }
+    catch (const hresult_error& ex)
+    {
+        hresult result{ ex.code() };
+        if (result == 0x80070002) // FileNotFoundException
```

Comment:
Can you use: `HRESULT_FROM_WIN32(ERROR_FILE_NOT_FOUND)` instead of a magic number?